libpromises/expand.c

```diff
@@ -632,7 +632,7 @@ void BundleResolvePromiseType(EvalContext *ctx, const Bundle *bundle, const char
             for (size_t i = 0; i < SeqLength(pt->promises); i++)
             {
                 Promise *pp = SeqAt(pt->promises, i);
-#if 1
+#if 0
```

Comment:
> Variable re-definition always occurs to my knowledge since years now. policy => "free/overrideable" has no effect. (though pre-eval is quite cloudy).

AFAICT variable redefinition had been always off during pre-eval.